File: object_oriented/ADT/Person.java
public class Person {
    String name;
    int age;
    
    Person(String name, int age) {
        this.name = name;
        this.age = age;
    }

    public String toString(){
        String ret = "Person: " +this.name + " is " +this.age+ " years old!";
        return ret;
    }
}

File: imperative/polymorphism/datastructures/Person.cpp
#include <iostream>
using namespace std;

class Person {
public:
    string name;
    int age;

    Person() {
        name = "";
        age = 0;
    }

    Person(string n, int a) {
        name = n;
        age = a;
    }

    bool operator<(const Person& other) const {
        return name < other.name;
    }
};